python_modules/dagster-graphql/dagster_graphql/schema/sensors.py

```diff
@@ -127,8 +127,9 @@ def __init__(
             else None,
         )
 
-    def resolve_id(self, _):
-        return self._external_sensor.get_external_origin_id()
+    def resolve_id(self, _) -> str:
+        # use semantic selector string for identifier
+        return self._external_sensor.selector.string
```

Comment:
I think this is fine?  When we need a schedule/sensor selector, I think we always provide a full selector input object, containing the location, repository, sensor / schedule names.

Will defer to @salazarm though, since I think id is used for the apollo cache and I'm not sure about the invalidation piece.